docs/nitpick-exceptions

```diff
@@ -37,32 +26,17 @@ py:class yaml.composer.Composer
 py:class yaml.resolver.Resolver
 py:class yaml.emitter.Emitter
 
-# astropy.units
-py:obj astropy.units.function.logarithmic.m_bol
```

Comment:
This one might still be needed for OSX and Windows because of case-insensitivity. RTD only builds on Linux but one may try to build doc locally on other OS too.

Reply:
I don't have access to Windows or macOS, so someone else will have to test this.

Reply:
I can try on Windows but not till next week. Thanks for your patience!

Reply:
On Windows with this branch, I see this warning, so I think you have to add it back. Thanks!

`astropy\docs\units/ref_api.rst:237: WARNING: autosummary: stub file not found 'astropy.units.function.logarithmic.m_bol'. Check your autosummary_generate setting.`